DistFiles/factoryCollections/Templates/Wall Calendar/configure.ts

```diff
@@ -43,11 +43,11 @@ class CalendarConfigurator {
     //
     updateDom():void {
         var year:string = this.configObject.year;
-        var originalMonthsPicturePage:HTMLDivElement = $('.calendarMonthTop')[0];
-        var pageToInsertAfter:HTMLDivElement = originalMonthsPicturePage;
+        var originalMonthsPicturePage:HTMLElement = $('.calendarMonthTop')[0];
+        var pageToInsertAfter:HTMLElement = originalMonthsPicturePage;
         for (var month:number = 0; month < 12; month++) {
 
-            var monthsPicturePage:HTMLDivElement = $(originalMonthsPicturePage).clone();
+            var monthsPicturePage:HTMLElement = $(originalMonthsPicturePage).clone()[0];
```

Comment:
Adding the [0] here feels like a substantive change. Probably good, but I had the impression you were just fixing up spurious warnings.